test/plugins/inlineStyles.07.svg

```diff
@@ -0,0 +1,17 @@
+<svg id="test" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100">
+    <style>
+        .red {
+            fill: red !important;
+        }
+        .blue {
+            fill: blue;
+        }
+    </style>
+    <rect width="100" height="100" class="blue red"/>
+</svg>
+
+@@@
+
+<svg id="test" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100">
+    <rect width="100" height="100" class="blue red" style="fill:red!important"/>
```

Comment:
When I run the `convertStyleToAttrs` plugin on this SVG, I get:

```
<svg id="test" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100">
  <path class="blue red" fill="red!important" d="M0 0h100v100H0z"/>
</svg>
```

Maybe this is a bug that should be fixed in a separate pull request? You probably know better than me. WDYT?